x/hub-genesis/keeper/keeper.go

```diff
@@ -46,20 +54,35 @@ func NewKeeper(
 		panic("mint keeper cannot be nil")
 	}
 
-	return Keeper{
+	sb := collections.NewSchemaBuilder(collcompat.NewKVStoreService(storeKey))
+
+	k := Keeper{
 		cdc:        cdc,
 		storeKey:   storeKey,
 		paramstore: ps,
 		ak:         ak,
 		bk:         bk,
 		mk:         mk,
+		gb:         gb,
+		OngoingChannels: collections.NewMap(
```

Comment:
seems like everywhere we use `Pending` and here `Ongoing` ? better to call it PendingChannels?